src/test/groovy/net/rubygrapefruit/platform/file/BasicFileEventFunctionsTest.groovy

```diff
@@ -617,16 +618,17 @@ class BasicFileEventFunctionsTest extends AbstractFileEventFunctionsTest {
             : Platform.current().windows
             ? [event(MODIFIED, removedFile), event(REMOVED, removedFile, false), event(REMOVED, watchedDir)]
             : [event(REMOVED, removedFile), event(REMOVED, watchedDir)]
-        removedDir.deleteDir()
+        def directoryDeleted = removedDir.deleteDir()
```

Comment:
Would be nice to standardize on "remove" as the term.

Reply:
Done.